openedx/core/djangoapps/user_authn/urls_common.py

```diff
@@ -66,14 +66,51 @@
 
     # logistration MFE flow
     url(r'^user_api/v1/account/password_reset/token/validate/$', password_reset.password_reset_token_validate,
-        name="user_api_password_reset_token_validate"),
+        name="user_api_password_reset_token_validate_legacy"),
 
     # logistration MFE reset flow
     url(
         r'^password/reset/(?P<uidb36>[0-9A-Za-z]+)-(?P<token>.+)/$',
         password_reset.password_reset_logistration,
         name='logistration_password_reset',
     ),
+
+    url(
+        r'^api/user/v1/account/registration/$',
+        register.RegistrationView.as_view(),
+        name="user_api_registration"
+    ),
```

Comment:
Can I suggest that for each of these urls, you place the new url right above the legacy url? It would improve readability.

Reply:
Thanks for the suggestion.